City-of-Helsinki/drupal-helfi-sote
# UHF-945: Redis configuration

To test this:

- `composer install`
- `docker compose pull` (or `docker-compose pull` if your're still using old docker compose)
- `make stop && make up`
- `drush cim`
- `drush cr`

Go to reports -> status report and make sure redis is working. It should report something like this:

![2022-02-04-09-53-04-screenshot](https://user-images.githubusercontent.com/771113/152492026-1f8e53b8-b625-46dd-9bca-cb244d68afb8.png)
